docs/diagrams/ParserManager.puml

```diff
@@ -0,0 +1,159 @@
+@startuml
```

Comment:
Guys the components in the UML diagram are a bit too small to decipher, will be great if you can make the diagram a bit more bigger.